bson/src/test/unit/util/JsonPoweredTestHelper.java

```diff
@@ -27,9 +27,20 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.URISyntaxException;
+import java.net.URL;
 import java.nio.charset.Charset;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.PathMatcher;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import static java.nio.file.Files.isDirectory;
+import static java.util.stream.Collectors.toMap;
```

Comment:
I'm not sure why this has been changed  - but I imagine its due to the merging.

Reply:
The changes introduce methods `testDir`/`testDocs`, which use the NIO.2 API, and are used in the new `ServerSelectionWithinLatencyWindowTest.java`.